api/core/rag/datasource/vdb/tidb_vector/tidb_vector.py

```diff
@@ -54,14 +55,14 @@ def _table(self, dim: int) -> Table:
         return Table(
             self._collection_name,
             self._orm_base.metadata,
-            Column("id", String(36), primary_key=True, nullable=False),
+            Column(Field.PRIMARY_KEY.value, String(36), primary_key=True, nullable=False),
             Column(
-                "vector",
+                Field.VECTOR.value,
                 VectorType(dim),
                 nullable=False,
                 comment="" if self._distance_func is None else f"hnsw(distance={self._distance_func})",
```

Comment:
The column comment part can be removed due to the `VECTOR INDEX idx_vector` part

Reply:
SGTM. Removed.